MoesifNet8Example/README.md

```diff
@@ -0,0 +1,64 @@
+# Moesif .Net 6 Example
```

Comment:
there are several places in this file where it needs to be changed from Net6 to Net8